embedded-bins/runc/Dockerfile

```
@@ -2,31 +2,36 @@ ARG BUILDIMAGE
 FROM $BUILDIMAGE AS build
 
 RUN apk add build-base git \
-	curl linux-headers gperf bash pkgconf
+	curl linux-headers gperf bash pkgconf patch cmd:tail
 
 ENV GOPATH=/go
 
 ARG LIBSECCOMP_VERSION=2.5.5
 RUN curl --proto '=https' --tlsv1.2 -L https://github.com/seccomp/libseccomp/releases/download/v$LIBSECCOMP_VERSION/libseccomp-$LIBSECCOMP_VERSION.tar.gz \
 	| tar -C / -zx
 
-RUN cd /libseccomp-$LIBSECCOMP_VERSION && ./configure --sysconfdir=/etc --enable-static
+WORKDIR /libseccomp-$LIBSECCOMP_VERSION
+# backport commit that lets us run the tests in parallel
+RUN curl -L https://github.com/seccomp/libseccomp/commit/2380f5788c692796f75e464c61aa877e5c4eb882.patch \
+	| patch -p1
+RUN ./configure --sysconfdir=/etc --enable-static
 
-RUN make -j$(nproc) -C /libseccomp-$LIBSECCOMP_VERSION
-RUN make -j$(nproc) -C /libseccomp-$LIBSECCOMP_VERSION check
-RUN make -C /libseccomp-$LIBSECCOMP_VERSION install
+RUN make -j$(nproc)
+RUN make -j$(nproc) check-build
+RUN cd tests && ./regression -j $(nproc)
```

I suppose we could use the standard check target and set this variable instead:

```suggestion
RUN make -j$(nproc) check LIBSECCOMP_TSTCFG_JOBS=$(nproc)
```

---

I think that was added in secondary git commit, and I trid to keep number of backported patches down to a minimal. I can "cherry-pick" those to if this is what we want.